test/nettests/utils.hpp

```diff
@@ -34,14 +34,6 @@ template <typename T> mk::nettests::BaseTest make_test(std::string s) {
     return make_test<T>().set_input_filepath("./test/fixtures/" + s);
 }
 
-static inline void run_async(mk::nettests::BaseTest test) {
-    volatile bool done = false;
-    test.start([&]() { done = true; });
-    do {
-        std::this_thread::sleep_for(std::chrono::seconds(1));
-    } while (!done);
-}
-
```

Comment:
No need to keep this abstraction since we've stopped running all tests both sync and async (see above comment)